cmd/porter/run_test.go

```diff
@@ -15,12 +15,14 @@ func TestRun_Validate(t *testing.T) {
 
 	p := porter.NewTestPorter(t)
 	p.TestConfig.SetupPorterHome()
-	p.TestConfig.TestContext.AddTestFile("../../templates/porter.yaml", config.Name)
+	configTpl, err := p.TestConfig.GetPorterConfigTemplate()
+	require.Nil(t, err)
```

Comment:
just a FYI, @vdice taught me about require.NoError and it's nice. When there is an error, it will print out the error contents. No need to change what you have here, just sharing what I learned. 😀